### DISCUSSION TITLE
I gave RTLSDR-Airband a shout out at DEF CON 30

### DISCUSSION THREAD

**Discussion by LoganSound** [rtl-airband/RTLSDR-Airband#336] [Show and tell]
We use RTLSDR-Airband to separate individual radio [@UFOsofDC](https://twitter.com/ufosofdc) for users to transcribe, they are then mapped with corresponding keywords and posted to the main [@HelicoptersofDC](https://twitter.com/HelicoptersofDC) account. [Check out the DEF CON 30 video cued up to our radio program.](https://youtu.be/KYuBf2HpXJg?t=922)

**Comment by szpajder** [comment]
Awesome, thank you!

I watched the whole thing. Well done! :-)
